homeassistant/components/ozw/climate.py

```diff
@@ -199,16 +200,18 @@ def fan_modes(self):
     @property
     def temperature_unit(self):
         """Return the unit of measurement."""
-        if self.values.temperature is not None and self.values.temperature.units == "F":
-            return TEMP_FAHRENHEIT
-        return TEMP_CELSIUS
+        return self._convert_units(self.values.temperature.units)
```

Comment:
We want to apply the suggestion in this comment:
https://github.com/home-assistant/core/issues/38579#issuecomment-693694852

This change doesn't seem to use the setpoint unit for temp unit.